samples/pmic/npmx/buck/README.rst

```diff
@@ -0,0 +1,101 @@
+This sample demonstrates the usage of nPM1300 PMIC BUCKs.
+For testing connect voltage multimeter to VOUT1 (output voltage pin of BUCK1). In this example
+voltage can be changed only for BUCK1. BUCK2 voltage is set by default to 3.0V.
+In main.c file select your test case to be executed.
+You can also modify expected output voltages and selected GPIOs to control the system.
+
+Testcases
+---------
+
+Test set output BUCK voltage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In `Testcase selection` select `Set output voltage (simple)`.
+Voltage is set to 2.4 V (`NPMX_BUCK_VOLTAGE_2V4`) on buck 1.
+
+Test output voltage range
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In `Testcase selection` select `Set output voltage (advanced)`.
+Now only test_output_voltage() function is called.
+BUCK1 is starting from voltage 1.0 V (`NPMX_BUCK_VOLTAGE_1V0`).
+The output voltage will increase from the selected to the maximum voltage (3.3 V).
+
+Test retention voltage using GPIO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In `Testcase selection` select `Set retention voltage of buck 1 with selected external pin`.
+Now only test_retention_voltage() function is called.
+GPIO1 (`NPMX_GPIO_INSTANCE_1`) is set to input mode and configured to be used as a retention input selector for buck 1 (`NPMX_BUCK_INSTANCE_1`).
+Normal mode voltage is set to 1.7 V (`NPMX_BUCK_VOLTAGE_1V7`) and retention voltage to 3.3 V (`NPMX_BUCK_VOLTAGE_3V3`).
+When you change the input state of the selected GPIO you should see voltage changes in the buck 1 output.
+
+Test enable BUCK using GPIO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In `Testcase selection` select `Set enable mode of buck 1 with selected external pin`.
+Now only test_enable_bucks() function is called.
+GPIO3 (`NPMX_GPIO_INSTANCE_3`) is set input mode and configured to be used as enable mode selector for buck 1 (`NPMX_BUCK_INSTANCE_1`).
+The output voltage of buck 1 is set to 3.3 V (`NPMX_BUCK_VOLTAGE_3V3`).
+When you change the input state of the selected GPIO you should see voltage changes in the buck 1 output - from 0 V to previously selected voltage.
+
+Requirements
+************
+
+To run this sample, do the following:
+
+* Connect the nRF5340 or nRF52840 DK to a host device.
+* Open a COM port.
+* Connect nPM1300 EK with the chosen DK.
+
+Connections
+-----------
+
+.. list-table:: nPM1300 EK connections.
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - nPM1300 EK
+     - nRF5340 DK
+     - nRF52840 DK
+   * - GPIO0
+     - P1.10
+     - P1.10
+   * - SDA
+     - P1.02
+     - P0.26
+   * - SCL
+     - P1.03
+     - P0.27
+   * - VOUT2 + GND
+     - External supply (P21)
+     - External supply (P21)
+
+On nPM1300 EK do the following:
+
+* Connect USB to power supply (J3).
+* On **P18** connect with the VOUT2 - VDDIO jumper.
+* On **P2** connect with the VBAT - VBATIN jumper.
+* On **P17** connect all LEDs with jumpers.
+* On **P13** connect with the RSET1 - GND jumper.
+* On **P14** connect with the RSET2 - VSET2 jumper.
+
+On nRF5340 DK do the following:
```

Comment:
As we have two options - one of two DKs to be chosen - I suggest creating tabs for them, similar to that:

![image](https://github.com/nrfconnect/sdk-nrf/assets/115017424/449997dc-cb18-4273-a590-e198996b03af)

If you agree, I'll do it.

Steps should be a little bit more descriptive - as above of EK.